styles.css

```diff
@@ -0,0 +1,547 @@
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: -1px;
+}
+
+.speakers1,
+.speakers2,
+.speakers3 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.speakeritems {
+  width: 100%;
+  display: flex;
+  margin: 10px;
+}
+
+.containerB {
+  padding: 10px;
+}
+
+.speakeritems img {
+  width: 107px;
+  height: 106.399px;
+}
+
+.itemsA {
+  padding: 5px 2px 20px 10px;
+}
+
+.itemsA h5 {
+  color: #000;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+  letter-spacing: -0.3px;
+}
+
+.itemsA span {
+  color: #e32116;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 13.943px;
+  letter-spacing: -0.1px;
+  margin-bottom: 20px;
+}
+
+.itemsA p {
+  color: #000;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 17.078px; /* 189.753% */
+  letter-spacing: -0.45px;
+  width: 229px;
+}
+
+.container5 img {
+  width: 200px;
+  height: 50.47px;
+}
+
+.section4 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  gap: 20px;
+  background: #414246;
+  justify-content: center;
+  height: 400px;
+}
+
+.container5 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.section4 h6 {
+  color: #8f8f91;
+  font-family: "Lato", sans-serif;
+  font-size: 30px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: -0.3px;
+  width: 98px;
+  height: 34px;
+}
+
+@media screen and (min-width: 768px) {
+  .header .header-list1 {
+    display: flex;
+    justify-content: end;
+    align-items: center;
+    background: #3e3e3e;
+    color: #fff;
+    gap: 10px;
+    padding: 1px 190px 1px 0;
+    font-size: 15px;
+    height: 30px;
+    font-weight: 400;
+  }
+
+  .containerA {
+    padding: 5px;
+    margin-left: 180px;
+    width: 70%;
+    height: 100%;
+    flex-shrink: 0;
+  }
+
+  .section1 {
+    display: flex;
+    height: 400px;
+    padding: 50px 0 0 35px;
+  }
+
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px 190px 0 0;
+    background: white;
+  }
+
+  .logo {
+    display: block;
+    width: 130px;
+    height: 51px;
+    margin-left: 190px;
+  }
+
+  .hambuger {
+    visibility: hidden;
+  }
+
+  .navbar ul {
+    display: flex;
+    background: white;
+    gap: 65px;
+    align-items: center;
+    flex-direction: row;
+    color: #868686;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 800;
+    line-height: normal;
+    position: static;
+    width: 56%;
+    visibility: visible;
+    padding: 0;
+  }
+
+  .campagne {
+    padding: 5px;
+    border: 5px solid #ec4327;
+    color: #ec4327;
+    font-family: "Lato", sans-serif;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;
+    width: 100%;
+  }
+
+  .displaynone {
+    display: none;
+  }
+
+  .section2 {
+    height: 700px;
+  }
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement according to which you can make little changes or adjustments of your own choice but can't change it completely but as I can see that your design looks different than actual, which makes your design `inconsistent` and looks unprofessional. Your mobile design `main program` card's height is even not consistent. Please have a look at the below-attached image and table.

### Requirement
![image](https://github.com/ulises2607/cayafate-project/assets/66476329/cd029716-65ab-45b5-8e7d-6886f05d48af)


### Desktop Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-22 16-23-30](https://github.com/ulises2607/cayafate-project/assets/66476329/1740fdb3-a789-41bb-a793-1b04a9133498)

</td>
<td>

![Screenshot from 2023-08-03 01-33-44](https://github.com/enocol/capstone/assets/66476329/c99b22a6-6409-4d81-8b12-63fec96434d3)

</td>
</tr>
</table>

### Mobile Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/ulises2607/cayafate-project/assets/66476329/821d8e2b-b333-4385-b85e-c7daa1b6ddba)

</td>
<td>

![image](https://github.com/enocol/capstone/assets/66476329/4e00fa74-fa63-446e-9841-7bc14345424f)

</td>
</tr>
</table>

Reply:
@skyv26 Thanks for your comment, but I just want to point out that you have made reference to the Figma design but the project requirement does not have any reference to a Figma design that you can copy a design from its styles properties to match the actual design. A  part of the requirement states that the website must be PERSONALISED and the design  provided servers ads a  GUIDELINE.

Reply:
@enocol Please note that I just mentioned to follow the design or adhere to the requirements but didn't mention to copy the design styles like CSS. I already mentioned that you have to follow the behance design not Figma design.

Reply:
@enocol I already attached the screenshot of the requirements, Please look at that carefully and if the requirements at your end are different from what I have attached, please let me know or share the link. I mentioned that the structure is inconsistent and if you carefully compare your design then you will find some problems or issues.